tests/autoblocks/test_run_test_suite.py

```diff
@@ -317,7 +317,7 @@ def test_error_in_evaluator(httpx_mock):
     def test_fn(test_case: MyTestCase):
         return test_case.input + "!"
 
-    class MySyncEvaluator(BaseTestEvaluator):
+    class MySyncEvaluator(BaseTestEvaluator[MyTestCase, str]):
```

Comment:
they'll need to specify generics for their evaluators like this in order to get their mypy checks to pass